src/counter/counterActions.ts

```diff
@@ -0,0 +1,9 @@
+import { createAsyncThunk } from "@reduxjs/toolkit";
+
+export const incrementAsync = createAsyncThunk(
```

Comment:
createAsyncThunk is something for handling API calls async. Not sure if we need this file right now. 

a counter action is not specced in the ticket.